README.md

```diff
@@ -1 +1,77 @@
+# NextJS Custom Menu App
 
+## URLs
+
+_Note: Instructions on how to fetch `menuIds` for a given API key are available in the "Multi-location support" section below._
+
+_Example menuId: 3404a063-2bbe-4473-bb1b-cac3c7e1d14e_
+
+**Localhost**
+
+- Main Menu Board: `http://localhost:3000?espressoAndCoffeeSubheading={espressoAndCoffeeSubheading}&espressoAndCoffeeBottomText={espressoAndCoffeeBottomText}&footnote={footnote}&menu={menuId}`
+
+**Production**
+
+<!-- - Main Menu Board: https://custom-menu-app-1.netlify.app?espressoAndCoffeeSubheading=<espressoAndCoffeeSubheading>&espressoAndCoffeeBottomText=<espressoAndCoffeeBottomText>&
+  footnote=<footnote>&menuId={menuId} -->
```

Comment:
I tried to setup the production deploy with Vercel, but can't seem to add this repo to Vercel. 

<img width="803" alt="Screen Shot 2022-07-28 at 2 48 22 pm" src="https://user-images.githubusercontent.com/1712435/181422936-1f704cee-7fcd-498d-82bd-8de2370a6879.png">

After clicking "Update access" the modal disappears but the repo does not appear on the list.